README.md

```diff
@@ -55,10 +55,10 @@ NativeUpdater.displayUpdateAlert(
   forceUpdate: true,
   appStoreUrl: '<Your App Store URL>',
   playStoreUrl: '<Your Play Store URL>',
-  titlePrefix: '<Your Title Prefix>',
-  description: '<Your Description>',
-  updateButtonLabel: '<Your Update Button Label>',
-  closeButtonLabel: '<Your Close Button Label>',
+  iOSDescription: '<Your iOS Description>',
```

Comment:
@JeffryHermanto just curious what about for the Android older version if we change this only for iOS needs?

Reply:
This is optional only for iOS, the Android version already handled with In-App Update.

Reply:
I have the same question as @haritsfahreza .

What happens when the app is installed in an older version that does not support InAppUpdates?

>In-app updates works only with devices running Android 5.0 (API level 21) or higher, and requires you to use Play Core library 1.5.0 or higher. Additionally, in-app updates support apps running on only Android mobile devices and tablets, and Chrome OS devices.

Reply:
If that's the case, the dialog will appear and show the exception message.